.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -262,15 +292,15 @@ public List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParseResult
             }
             else
             {
-                slot.Value = DateTimeResolution(slot, false, false, false);
-                slot.Type = $"{ParserTypeName}.{DetermineDateTimeType(slot.Type, false, false, false)}";
+                slot.Value = DateTimeResolution(slot);
+                slot.Type = $"{ParserTypeName}.{DetermineDateTimeType(slot.Type, false)}";
```

Comment:
Please pass named parameters for values with no context.

Reply:
Fixed.